server/polar/license_key/endpoints.py

```diff
@@ -0,0 +1,153 @@
+from fastapi import Depends
+from pydantic import UUID4
+
+from polar.authz.service import AccessType, Authz
+from polar.exceptions import ResourceNotFound, Unauthorized
+from polar.kit.db.postgres import AsyncSession
+from polar.kit.pagination import ListResource, PaginationParamsQuery
+from polar.models import LicenseKey, LicenseKeyActivation
+from polar.openapi import APITag
+from polar.organization.service import organization as organization_service
+from polar.postgres import get_db_session
+from polar.routing import APIRouter
+
+from . import auth
+from .schemas import (
+    LicenseKeyActivationRead,
+    LicenseKeyRead,
+    LicenseKeyUpdate,
+    LicenseKeyWithActivations,
+    NotFoundResponse,
+    UnauthorizedResponse,
+)
+from .service import license_key as license_key_service
+
+router = APIRouter(prefix="/license-keys", tags=[APITag.documented, APITag.featured])
```

Comment:
🚨 You need to add a dedicated tag first, like `license_keys`:

```
tags=["license_keys", APITag.documented, APITag.featured]
```

Otherwise, generated JS SDK and documentation will name the group "Documented" (see next comment)